pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Start nettop monitoring TCP sockets from Loopback interfaces:
+
+`nettop -m tcp -t loopback`
+
+- Start nettop monitoring specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Start nettop displaying summary only for per-process:
+
+`nettop -P`
+
+- Start nettop in logging mode and display 10 samples:
+
+`nettop -l {{10}}`
+
+- Start nettop in delta mode, updating every 5 seconds:
```

Comment:
What are `logging` and `delta` mode exactly?

Reply:
`delta` mode will show delta value between every update rather than show cumulative value.

`loggin` mode will print the raw form periodically rather than redisplay.

I think man page in https://www.manpagez.com/man/1/nettop/ is outdated so it dones't have these options.
I pasted the result of `man nettop` to here: https://pastebin.com/vqJc1et3, hope you cloud check it if it is convenient :)

Reply:
I'm not sure if that's the case for OSX, but if you can download that package it has to exist somewhere online to browse through. These files probably contain also the manpage. Maybe that's worth a try.

Reply:
It's an OSX builtin command. Sorry, according to this answer: https://stackoverflow.com/a/8670402, this command use private code, maybe we can't find the original online man page, just like `launchctl` and `hdiutil`.

Reply:
https://github.com/tldr-pages/tldr/pull/6254/files#r673909261

Reply:
OK, I see it 😄